manifests/desktop.pp

```diff
@@ -37,6 +41,12 @@
     }
   }
 
+  exec { 'dconf update':
```

Comment:
The simp/dconf module is already in the control repo `Puppetfile`.  Perhaps this module could depend directly on that? https://forge.puppet.com/modules/simp/dconf/readme

Reply:
Alright, I pushed a new version that uses Somebody Else's Code to do the same thing.
As usual, the tests now fail in an opaque way.

Reply:
I managed to test the tests to pass.